pymongo/topology.py

```diff
@@ -187,16 +187,18 @@ def select_servers(self,
 
     def _select_servers_loop(self, selector, timeout, address):
         """select_servers() guts. Hold the lock when calling this."""
+        selectors = [selector, self._settings.server_selector]
+
         now = _time()
         end_time = now + timeout
-        server_descriptions = self._description.apply_selector(
-            selector, address)
+        server_descriptions = self._description.apply_selectors(
+            selectors, address)
```

Comment:
I think it would be clearer to pass the custom server selector as a another argument instead of a list.